appservice/src/createAppService/IAppServiceWizardContext.ts

```diff
@@ -72,4 +73,16 @@ export interface IAppServiceWizardContext extends IResourceGroupWizardContext, I
      */
 
     recommendedSiteRuntime?: LinuxRuntimes[];
+
+    /**
+     * Application insights components are necessary for Function apps log streaming.  By default, we should instance
```

Comment:
"we should instance one"

Reply:
I'm not sure what this comment means.  Should I move "one" up a line?